data/campaigns/World_Conquest/lua/campaign/scenario.lua

```diff
@@ -66,9 +66,11 @@ end)
 -- NOTE: this is a bit fragile, in particualr it breaks if difficulty_selection happens before the prestart event above.
 on_event("wc2_start", function(cx)
 	for side_num = 1, wml.variables.wc2_player_count do
-		wesnoth.wml_actions.wc2_start_units {
-			side = side_num
-		}
+		if wml.variables.wc2_scenario == 1 then
+			wesnoth.wml_actions.wc2_start_units {
+				side = side_num
+			}
+		end
```

Comment:
Either way works and there's no practical problem here, but I think doing `if ... then; for ... do` would be better style than `for ... do; if ... then`, because the condition being checked is the same each time through the loop.